crates/hc/src/dna.rs

```diff
@@ -1,18 +1,61 @@
 //! Helpers for working with dna files.
 use std::path::Path;
 use std::path::PathBuf;
+use std::process::Stdio;
 
 use anyhow::anyhow;
 use anyhow::ensure;
+use structopt::StructOpt;
+use tokio::process::Command;
 use walkdir::WalkDir;
 
+mod util;
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "dna-util")]
```

Comment:
What does this line do? Does it mean you have to use `hc dna-util`? I think it could just be `hc dna`, since it is a subcommand of an existing "util".

Reply:
That's just copy and pasted from the dna-util crate. It seems to have no effect so it can probably just be removed.
The command is just `hc dna`. Ideally no one will need to really use these command directly but we're not there for all directory layouts yet.